docs/user_guides/fs/feature_group/notification.md

```diff
@@ -0,0 +1,100 @@
+---
+description: Documentation on Change Data Capture for feature groups in Hopsworks.
+---
+
+# Change Data Capture for a Feature Group
+
+## Introduction
+
+Changes to online-enabled feature groups can be captured by listening to events on specified topics.
+This optimizes the user experience by allowing users to proactively make predictions as soon as there is an update on the features.
+
+In this guide you will learn how to enable Change Data Capture (CDC) for online feature groups within Hopsworks, showing examples in HSFS APIs as well as the user interface.
+
+## Prerequisites
+
+Before you begin this guide we suggest you read the [Feature Group](../../../concepts/fs/feature_group/fg_overview.md) concept page to understand what a feature group is and how it fits in the ML pipeline.
+Subsequently [create a Kafka topic](../../projects/kafka/create_topic.md), this topic will be used for storing Change Data Capture events.
+
+## Using HSFS APIs
+
+### Create a Feature Group with Change Data Capture
+
+To enable Change Data Capture for an online-enabled feature group using the HSFS APIs you need to [create a feature group](./create.md) and set the `notification_topic_name` properties value to the previously created topic.
+
+=== "Python"
+
+    ```python
+    fg = fs.create_feature_group(
+      name="feature_group_name",
+      version=feature_group_version,
+      primary_key=feature_group_primary_keys,
+      online_enabled=True,
+      notification_topic_name="notification_topic_name")
+    ```
+
+### Update Feature Group Change Data Capture topic
+
+The notification topic name can be changed after the creation of the feature group.
+By setting the `notification_topic_name` value to `None` or empty string notification will be disabled.
+With the default configuration, it can take up to 30 minutes for these changes to take place since the onlinefs service internally caches feature groups.
```

Comment:
in onlinefs it is set like this: `.expireAfterAccess(30, TimeUnit.MINUTES)`
so user would not be able to change it (same for all cached entries), should we enable user changing this value?

Reply:
also it might be better to change them from `expireAfterAccess` (this includes last access, so if user constantly writes it might not expire ever) to `expireAfterWrite`

Reply:
Yeah It should be a configurable parameter.